### DISCUSSION TITLE
How to implement EffectCreative in Swiper 6?

### DISCUSSION THREAD

**Discussion by ktim816** [nolimits4web/swiper#4913] [Q&A]
Hey! I would like to know how I can implement EffectCreative in Swiper 6, is this possible?

**Comment by vltansky** [comment]
Just curious what is the reason you cant migrate to Swiper 7?

**Reply by ktim816** [reply]
> Just curious what is the reason you cant migrate to Swiper 7?

Our project includes both ESM and CommonJS modules, so now we cannot fully migrate to ESM modules.
